second_response_to_reviewers.tex

```diff
@@ -0,0 +1,366 @@
+literature. It can be thought of as a tournament on Moran processes equivalent
+to the seminal work by Robert Axelrod in the 1980s.
+
+Robert Axelrod's work on the Prisoner's Dilemma tournaments (described in
+the literature review) which has been cited more than 34,000 times considered
+pair wise matches between strategies. Our work here provides a much more robust
+type of tournament where strategies are pitted against one another in an actual
+evolutionary process (as opposed to a single one of interaction).
+
+\section{Comments from first referee}
+
+\begin{verbatim}
+   Figures are much improved! It was a good idea to move some of them to the
+Supplementary Material. I still think the title is not representative of the
+work -- you should reconsider at least changing the subtitle. You are not
+studying the Moran process; you are studying Prisoner's Dilemma strategies.  I
+\end{verbatim}
+
+Thank you for agreeing to review the paper again.
+
+We have changed the subtitle to be: `Empirical Study of strategies in the Moran
+process for the Iterated Prisoner's
+
+\begin{verbatim}
+also still think the description on page 3 of how you found the TF1, TF2, and
+TF3 strategies is confusing. It has a frustrating level of detail. You should
+either have less detail or more detail. This is not a crucial point, however.
+\end{verbatim}
+
+The description of how we obtain the strategies is quite minimalistic.
+The rest of the description in this section is specifically describing each
+strategy (accompanied by the corresponding diagram). This description is quite
+important as it highlights the handshake mechanism which is one of the main
+findings of the work. We have considered changing this further but as this is
+not a crucial point we would prefer to leave it as it is.
+
+\section{Comments from second referee}
+
+\begin{verbatim}
+
+I couldn’t tell how each round is played. Does each individual play against only
+one
+other (randomly chosen) individual or against all N − 1 other individuals?
+
+\end{verbatim}
+
+Yes they play against all N - 1 other individuals. This is a standard Moran
+process of this type and is described in
+Algorithm 2. The mathematical equations (2) and (3) also ensures this to be the
+case. Nonetheless we have added a sentence clarifying this:
+
+``Individuals are \textbf{selected}
+according to a given fitness landscape, in this work it is their total utility
+against all other individuals in the population.''
+
+\begin{verbatim}
+    In the introduction the authors state "The results and insights contained in
+    this paper
+    would be difficult to derive analytically”. But in Section II the authors
+    present several
+    formulas (Eq. (2)–(7)) and analytically compare a couple of strategies. Why
+    can’t
+    these formulas be used to analytically compare any pair of strategies listed
+    in the
+    paper?
+\end{verbatim}
+
+This is the purpose of section II. The analytical results hold for deterministic
```

Comment:
The purpose of this section is to show that the simulations match the theory well in cases that are analytically accessible as a validation. This is in addition to the extensive test suite for the software which covers 100% of the Axelrod library.

As an example of a case where a theoretical result is more difficult to obtain, we note that when one or more of the strategies is stochastic then the utility function is also stochastic and would need to be analyzed statistically, assuming that the utility distribution could be analytically derived, which is unrealistic in many cases. Instead we approximate the utilities with a large number of simulations, a commonly used technique.